data-raw/process_data_set_jessica.R

```diff
@@ -0,0 +1,16 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+nbaallelo <- read_csv("data-raw/nba-elo/nbaallelo.csv")%>%
+  arrange(game_id, `_iscopy`)%>%
+  mutate_if(is.character,as.factor)%>%
+  mutate(opp_win_equiv = lead(win_equiv), opp_seasongame = lead(seasongame, 1),
```

Comment:
Could you put a line break between each variable that's being mutated?  See https://style.tidyverse.org/pipes.html#long-lines-2